README.md

```diff
@@ -352,18 +354,36 @@ buildctl build ... --import-cache type=local,src=path/to/input-dir
 
 The directory layout conforms to OCI Image Spec v1.0.
 
+#### GitHub Actions cache
+
+```bash
+buildctl build ... \
+  --output type=image,name=docker.io/username/image,push=true \
+  --export-cache type=gha \
+  --import-cache type=gha
+```
+
+Following attributes are required to authenticate against the [Github Actions Cache service API](https://github.com/tonistiigi/go-actions-cache/blob/master/api.md#authentication):
+* `url`: Cache server URL (default `$ACTIONS_CACHE_URL`)
+* `token`: Access token (default `$ACTIONS_RUNTIME_TOKEN`)
+
+:information_source: This cache should be used exclusively in a [GitHub Action](https://github.com/features/actions)
```

Comment:
link to https://github.com/docker/build-push-action instead of https://github.com/features/actions

.. cache `url` and `token` automatically. To use this backend in inline Github workflow step https://github.com/crazy-max/ghaction-github-runtime needs to be used.

Reply:
Done